code/modules/cm_aliens/structures/trap.dm

```diff
@@ -217,6 +217,20 @@
 			to_chat(X, SPAN_XENONOTICE("[src] is occupied by a child."))
 			return XENO_NO_DELAY_ACTION
 
+	if(X.a_intent == INTENT_HARM)
+		to_chat(X, SPAN_XENONOTICE("You start tearing away at the hole."))
+		xeno_attack_delay(X)
```

Comment:
I don't think there's really any point to add an attack delay before a do_after.